apps/main/src/loan/components/PageLoanCreate/LoanFormCreate/components/DetailInfoTradeRoutes.tsx

```diff
@@ -23,18 +33,61 @@ const DetailInfoTradeRoutes = ({
   outputSymbol: string
   routes: string
 }) => {
-  const parsedRoutes = useMemo(() => {
-    if (routes) {
-      const delimiterPattern = /\s*-->\s*|\s*->\s*/
-      return routes.split(delimiterPattern)
-    } else {
-      return []
-    }
-  }, [routes])
-
+  const [releaseChannel] = useReleaseChannel()
+  const parsedRoutes = useMemo(() => routes?.split(/\s*-->\s*|\s*->\s*/) || [], [routes])
   const routesLength = parsedRoutes.length
-  const isMultiRoutes = routesLength > 1
-  const detailRowJustifyContent = isMultiRoutes ? 'flex-start' : 'flex-end'
+  const isMultiRoutes = parsedRoutes.length > 1
+
+  if (releaseChannel === ReleaseChannel.Beta) {
+    return (
+      <ActionInfo
+        label={t`Trade routed through:`}
+        value={
+          parsedRoutes?.length ? (
+            <WithWrapper wrap={isMultiRoutes} Wrapper={Stack} direction="row" gap={Spacing.sm}>
+              {isMultiRoutes && <RouteTrack />}
+              <WithWrapper wrap={isMultiRoutes} Wrapper={Stack} width="100%">
+                {parsedRoutes.map((route, i) => {
+                  const from = !i && input && `${formatNumber(input)} ${inputSymbol}`
+                  const to =
+                    i === parsedRoutes.length - 1 && output && `receive ${formatNumber(output)} ${outputSymbol}`
+                  return (
+                    <Stack key={route} direction="row" justifyContent="space-between" title={route}>
+                      <Typography
+                        sx={{
+                          maxWidth: 120,
+                          overflow: 'hidden',
+                          textOverflow: 'ellipsis',
+                        }}
+                        variant="bodySBold"
+                      >
+                        {route}
+                      </Typography>{' '}
+                      <Typography variant="bodySRegular">{(isMultiRoutes && from) || to}</Typography>
+                    </Stack>
+                  )
+                })}
+              </WithWrapper>
+            </WithWrapper>
+          ) : (
+            '-'
+          )
+        }
+        {...(parsedRoutes && {
+          alignItems: 'start',
+          sx: {
+            flexWrap: 'wrap',
+            // for some reason the routes are not taking 100% width, and we need to override the ActionInfo styles
```

Comment:
If you wanna try get rid of these 100%s I don't really understand why we need them rn 🤔